stdlib/Unicode/src/Unicode.jl

```diff
@@ -143,6 +143,69 @@ letter combined with an accent mark is a single grapheme.)
 """
 graphemes(s::AbstractString) = Base.Unicode.GraphemeIterator{typeof(s)}(s)
 
+"""
+    graphemes(s::AbstractString, m:n) -> `SubString`
```

Comment:
```suggestion
    graphemes(s::AbstractString, m:n) -> SubString
```
The line is already rendered as code, so the extra ` can be removed.